front/tests/pages/stdcm-page-model.ts

```diff
@@ -477,24 +573,20 @@ class STDCMPage {
     });
   }
 
-  async clickOnAllViaButton() {
+  async displayAllOperationalPoints() {
     await this.allViasButton.click();
   }
 
-  async clickOnRetainSimulation() {
-    await this.retainSimulationButton.click();
-  }
-
   async retainSimulation() {
-    await this.clickOnRetainSimulation();
+    await this.retainSimulationButton.click();
     await expect(this.downloadSimulationButton).toBeVisible();
     await expect(this.downloadSimulationButton).toBeEnabled();
     await expect(this.startNewQueryButton).toBeVisible();
   }
 
   async downloadSimulation(browserName: string) {
     // Wait for the download event
-    await this.page.waitForTimeout(500);
+    await this.page.waitForTimeout(1000);
```

Comment:
can't we use [expect.poll() and expect.toPass()](https://playwright.dev/docs/test-assertions#expectpoll) for those kind of situations?

I think we could avoid many waitForTimeout this way.

Reply:
Done